netbox/dcim/svg/cables.py

```diff
@@ -378,86 +348,101 @@ def render(self):
-                    if fanout:
-                        self.cursor += FANOUT_HEIGHT
-                        terminations = self.draw_terminations(far_ends)
-                        for term in terminations:
-                            if hasattr(term.object, 'cable') and link_cables.get(term.object.cable.pk):
-                                self.draw_fanout(term, link_cables.get(term.object.cable.pk))
-                    else:
-                        self.draw_terminations(far_ends)
-                elif far_ends:
-                    self.draw_terminations(far_ends)
-                else:
-                    # Link is not connected to anything
-                    break
-
-                # Far end parent
-                parent_objects = set(end.parent_object for end in far_ends)
-                self.draw_parent_objects(parent_objects)
+
+                parent_object_nodes, far_terminations = self.draw_far_objects(set(end.parent_object for end in far_ends), far_ends)
+                for cable in links:
+                    # Fill in labels and description with all available data
+                    description = [
+                        f"Link {cable}",
+                        cable.get_status_display()
+                    ]
+                    near = []
+                    far = []
+                    color = '000000'
+                    if cable.description:
+                        description.append(f"{cable.description}")
+                    if cable is Cable:
+                        labels = [f"{cable}"] if len(links) > 2 else [f"Cable {cable}", cable.get_status_display()]
+                        if cable.label:
+                            description.append(f"NetBox ID: {cable.id}")
+                        if cable.type:
+                            description.append(f"Type: {cable.get_type_display()}")
+                        if cable.length:
+                            description.append(f"Length: {cable.length} {cable.length_unit}")
+                        color = cable.color or '000000'
+
+                        # Collect all connected nodes to this cable
+                        near = [term for term in near_terminations if term.object in cable.a_terminations]
+                        far = [term for term in far_terminations if term.object in cable.b_terminations]
+                    elif cable is WirelessLink:
+                        labels = [f"{cable}"] if len(links) > 2 else [f"Wireless {cable}", cable.get_status_display()]
+                        if cable.ssid:
+                            description.append(f"SSID: {cable.ssid}")
+                        if cable.auth_type:
+                            description.append(f"AuthType: {cable.auth_type}")
+                        if cable.auth_cipher:
+                            description.append(f"AuthCipher: {cable.auth_cipher}")
+                        if cable.auth_psk:
+                            description.append(f"PSK is set")
```

Comment:
None of the extra data is in scope for these BR's.

Reply:
Removed those, except ssid (has been there before)